meeting_minutes/2025/2025-06-25.md

```diff
@@ -0,0 +1,90 @@
+![image](https://user-images.githubusercontent.com/1690898/139102180-5c1e2583-14f1-4f58-ab2b-9e3807ed529c.png)
+
+# Common Security Advisory Framework (CSAF) Technical Committee Working Meeting
+
+- Meeting Date: June 25, 2025
+- Time: 18:00 UTC (19:00 CET, 13:00 EST, 10:00 PST)
+
+## Call to Order and Welcome
+
+- Meeting called to order.
+
+## Roll call
+
+- Using OQC for roll call and quorum calculation.
+
+## Participants
+
+| Given Name | Family Name | Affiliation                                                 | Role          | Present |
+|:-----------|:------------|:------------------------------------------------------------|:--------------|:--------|
+| Christian  | Banse       | Fraunhofer-Gesellschaft e.V. - Fraunhofer AISEC             | Voting Member | No      |
+| Christoph  | Plutte      | Ericsson AB                                                 | Voting Member | Yes     |
+| Denny      | Page        | Individual                                                  | Voting Member | No      |
+| Dina       | Truxius     | Federal Office for Information Security (BSI)               | Voting Member | Yes     |
+| Feng       | Cao         | Oracle                                                      | Member        | No      |
+| JD         | Stefaniak   | Dell                                                        | Voting Member | No      |
+| Justin     | Murphy      | DHS Cybersecurity and Infrastructure Security Agency (CISA) | Co-Chair      | Yes     |
+| Martin     | Prpic       | Red Hat                                                     | Voting Member | Yes     |
+| Michael    | Reeder      | Dell                                                        | Voting Member | Yes     |
+| Omar       | Santos      | Cisco Systems                                               | Co-Chair      | Yes     |
+| Sergii     | Demianchuk  | Cisco Systems                                               | Member        | Yes     |
+| Sonny      | van Lingen  | Huawei Technologies Co., Ltd.                               | Voting Member | Yes     |
+| Stefan     | Hagen       | Individual                                                  | Co-Chair      | Yes     |
+| Thomas     | Proell      | Siemens AG                                                  | Member        | No      |
+| Thomas     | Schaffer    | Cisco Systems                                               | Member        | No      |
+| Thomas     | Schmidt     | Federal Office for Information Security (BSI)               | Voting Member | Yes     |
+| Tobi       | Limmer      | Siemens AG                                                  | Voting Member | No      |
```

Comment:
According to my list(s) and OQC Tobias Limmer does not have voting rights (was not present) the last 3 meetings.
```suggestion
| Tobi       | Limmer      | Siemens AG                                                  | Member        | No      |